test/vfd_swmr_group_writer.c

```diff
@@ -3109,37 +5450,39 @@ main(int argc, char **argv)
      * If the named pipe is not forced to communicate in every step, the reader may go ahead
      * to verify the group and the attribute operations before the writer has a chance to
      * carry out the corresponding operations. */
-    if (s.at_pattern != ' ') {
-        s.attr_test = true;
-        if (s.use_named_pipes)
-            s.csteps = 1;
-    }
+    if (s.attr_test && s.use_named_pipes)
+        s.csteps = 1;
+
+    /* For group operation test, force the named pipe to communicate in every step. */
+    if (s.grp_op_test && s.use_named_pipes)
+        s.csteps = 1;
```

Comment:
I think s.attr_test and s.grp_op_test cover all the test cases.  When will s.csteps be not set to 1?

Reply:
The default group test is the case when "s.attr_test and s.grp_op_test both false". For that test, s.csteps is not set to 1.